delivery_carrier_label_ups/models/stock_picking.py

```diff
@@ -13,6 +13,19 @@
 class StockPicking(models.Model):
     _inherit = "stock.picking"
 
+    @api.depends(
+        'delivery_type', 'location_id.usage', 'carrier_tracking_ref', 'state'
+    )
+    def _compute_show_label_button(self):
```

Comment:
A docstring mentioning `returns` will be appreciated I think.

Reply:
done